botocore/response.py

```diff
@@ -45,6 +46,12 @@ def __init__(self, raw_stream, content_length):
         self._content_length = content_length
         self._amount_read = 0
 
+    def __del__(self):
+        #Extending destructor in order to preserve the underlying raw_stream.
```

Comment:
"Stubbing destructor in order to..." may be more accurate here but that's a pretty minor nit.

We can probably remove the info about __del__ after this sentence since it'll be safe to assume users digging this deep are familiar.

Last nit, it's best practice to have a space between the # and the first character for comments. After that I think we're good to go here.